client/src/pages/CreateNewPage.css

```diff
@@ -0,0 +1,46 @@
+.Form {
+}
+
+.Page__heading {
+  border: 2px none black;
+  padding: 2em 8em;
+  border-radius: 15px;
+  background: #f4b7b7;
+  display: flex;
+  align-items: center;
```

Comment:
if you want to center the text horizontically you also need `justify-content: center`